### ISSUE TITLE
Update docs for target desktops(Ubuntu? Unity? GNOME?)

### ISSUE THREAD

**Issue by paravoid** [yktoo/indicator-sound-switcher#77]
README.md:
> Sound input/output selector indicator for **Ubuntu/GNOME**.
> It shows an icon in the **GNOME Panel**.

man/indicator-sound-switcher.1:
> is a sound input/output selector indicator for **Ubuntu/Unity**.
> It shows an icon in the indicator area in **Ubuntu's Panel**.

debian/control:
> A simple yet usable application indicator for **Ubuntu/Unity**.

These are clearly inconsistent (it seems that the Unity references were dropped at some point, but only from README.md). But, also, none of these are accurate either. I've been using this on my Debian MATE desktop which is neither Ubuntu, Unity, nor GNOME :)

**Comment by yktoo** [comment]
Thanks @paravoid. Indeed, it all started as an indicator app for the now deceased Unity, then Canonical released a compatibility layer for GNOME and, apparently, its friends.

I'm not really aware of what DEs that compatibility layer supports, perhaps you can advice on that?

**Comment by paravoid** [comment]
I'm no expert myself but: you seem to be using the AppIndicator3 API. `apt show gir1.2-appindicator3-0.1` (or `apt show libappindicator3-1`) says:
> A library to allow applications to export a menu into the panel.
> Based on KSNI it also works in KDE and will fallback to generic Systray
> support if none of those are available.

So, I think it should work on all freedesktop.org-compatible desktops!

Also, note that  there seems to be a better-maintained forked of application indicators etc. under the name of Ayatana. See [Ayatana Indicator's home page](https://ayatanaindicators.github.io/) as well as the email to [debian-devel email from last year](https://lists.debian.org/debian-devel/2018/03/msg00506.html).

(Again, not an expert, this is just AIUI).

**Comment by paravoid** [comment]
With regards to Ayatana (and the email thread above), I pushed paravoid/indicator-sound-switcher@1f62ccc27bbdcd856a998d75b04fc2a464fb54cc just now to add support for Ayatana. For simplicity I made it depend on #76 for now.

For the purposes of this issue... this should work on every desktop that supports [SNI](https://www.freedesktop.org/wiki/Specifications/StatusNotifierItem/), and I think it will also fall back on the generic [system tray spec](https://www.freedesktop.org/wiki/Specifications/systemtray-spec/). Basically... all desktops :)
